### PR TITLE
I have correct the homework errors

### PR DESCRIPTION
**Pull request by alsafi25**
# Your Details

- Your Name: KHALID ELSAFI
- Your City: NOTTINGHAM
- Your Slack Name:Khalid Elsafi

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Comment by Abdulqadir90** [comment]
Q17 and 18 needs updating i think. Lovely work Khalid

File: homework.md [hunk 1]
```diff
@@ -4,38 +4,38 @@
...
 
-Hyper Tall Massive Language
+Hyper Text Markup Language.
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Cascading Style Sheets
```
Comment (Abdulqadir90) [inline homework.md:31]: Hey Khalid, perhaps you could add more info why CSS is useful.

**Comment by alsafi25** [comment]
Ok I will do that

File: homework.md [hunk 2]
```diff
@@ -44,37 +44,38 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a text-based programming language used both on the client-side and server-side that allows you to make web pages interactive.
 
 ## Question 8
 
 What three problem does Git & Github solve?
```
Comment (remidoukmak) [inline homework.md:51]: Hi Khalid, try to write full sentences when adding the three problems. Answers sounds fine.

File: homework.md (hunk 1 above)
Comment (alsafi25) [reply homework.md:31]: ok that fine I will try to add more info

File: homework.md [hunk 3]
```diff
@@ -86,19 +87,19 @@ You send the changes of your local repository to Github
...
 
-Send you code to Github
+Commit the file means that it should be tracked in source control.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio Code.
```
Comment (Waleedip) [inline homework.md:102]: In regard to question #18, you could have added more sources of help on git

File: homework.md (hunk 2 above)
Comment (alsafi25) [reply homework.md:51]: Thanks I will update

**Pull request closed without merging**